src/main/java/uk/gov/register/presentation/view/RecordView.java

```diff
@@ -43,4 +48,17 @@ public ObjectNode getRecordJson() throws JsonProcessingException {
     public Map<String, FieldValue> getContent() {
         return record.item.getFieldsStream().collect(Collectors.toMap(Map.Entry::getKey, itemConverter::convert));
     }
+
+    @Override
+    public CsvSchema csvSchema() {
+        CsvMapper csvMapper = new CsvMapper();
+        csvMapper.disable(MapperFeature.SORT_PROPERTIES_ALPHABETICALLY);
+        CsvSchema entrySchema = csvMapper.schemaFor(Entry.class);
+
+        CsvSchema.Builder schemaBuilder = entrySchema.rebuild();
+        for (String value : getRegister().getFields()) {
```

Comment:
this method is now repeated in 3 or 4 separate places. is it time to remove this duplication somehow?